packages/labs/gen-manifest/goldens/test-element-a/custom-elements.json

```diff
@@ -59,6 +60,16 @@
           ],
           "tagName": "element-a",
           "customElement": true,
+          "attributes": [
+            {
```

Comment:
This should also have `foo`.

Reply:
I updated the golden with attributes and with new formatting. the formatting introduces unrelated whitespace and key-ordering changes, but those changes to help a lot with testing